docs/ProductionChecklistForScalarDBCluster.md

```diff
@@ -0,0 +1,73 @@
+# Production checklist for ScalarDB Cluster
+
+This checklist provides recommendations when deploying ScalarDB Cluster in a production environment.
+
+## Before you begin
+
+In this checklist, we assume that you are deploying ScalarDB Cluster on a managed Kubernetes cluster, which is recommended.
+
+## Production checklist: ScalarDB Cluster
+
+The following is a checklist of recommendations when setting up ScalarDB Cluster in a production environment.
+
+### Number of pods and Kubernetes worker nodes
+
+To ensure that the Kubernetes cluster has high availability, you should use at least three worker nodes in three availability zones and deploy at least three pods. You also should ensure that one pod is deployed on one worker node by using [Node Affinity](https://kubernetes.io/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity/).
+
+### Worker node specifications
+
+From the perspective of commercial licenses, resources for one pod running ScalarDB Cluster are limited to 2vCPU / 4GB memory. In addition, we recommend deploying one ScalarDB Cluster pod and one Envoy pod on one worker node. Note that you do not need to deploy an Envoy pod when using `direct-kubernetes` mode.
+
+In other words, the following components run on one worker node:
+
+* ScalarDB Cluster pod (2vCPU / 4GB)
+* Envoy proxy (0.2–0.3 vCPU / 256–328 MB)
+* Kubernetes components
+
+With this in mind, you should use a worker node that has 4vCPU / 8GB memory resources. We recommend running only the above components on the worker node for ScalarDB Cluster.
+
+### Network
+
+You should create the Kubernetes cluster on a private network since ScalarDB Cluster does not provide any services to users directly via internet access. We recommend accessing ScalarDB Cluster via a private network from your applications.
+
+### Monitoring and logging
+
+You should monitor the deployed components and collect their logs. For details, see [Monitoring Scalar products on a Kubernetes cluster](./K8sMonitorGuide.md) and [Collecting logs from Scalar products on a Kubernetes cluster](./K8sLogCollectionGuide.md).
+
+### Backup and restore
+
+You should enable the automatic backup feature and PITR feature in the backend database. For details, see [Set up a database for ScalarDB/ScalarDL deployment](./SetupDatabase.md).
+
+## Production checklist: Client application that uses ScalarDB Cluster
+
+The following is a checklist of recommendations when setting up a client application that is using ScalarDB Cluster in a production environment.
+
+### Transaction manager configuration
+
+You must always run CRUD requests via ScalarDB Cluster. To ensure requests are running properly, check the properties file for your client application and confirm that `scalar.db.transaction_manager=cluster` is configured.
```

Comment:
We also have SQL interface in ScalarDB Cluster actually. Should we also mention it?

Reply:
Ah, I see. You're right. We should mention it.
I will add the description about `scalar.db.sql.connection_mode=cluster`.

Reply:
I added the new section `SQL connection configuration` in 7859246e0143d3b9f15c451a8abe2cd30f1f1c3c.